interpreter/cling/lib/Interpreter/InvocationOptions.cpp

```diff
@@ -92,7 +92,6 @@ static const char kNoStdInc[] = "-nostdinc";
 
   static void ParseLinkerOpts(cling::InvocationOptions& Opts,
                               InputArgList& Args /* , Diags */) {
-    Extend(Opts.LibsToLoad, Args.getAllArgValues(OPT_l));
```

Comment:
This change reasons that `-l` is less useful than `-load`. I would argue that
- `-l` is much more user-centric and much more common,
- `-L` without `-l` is surprising.

I'd be in favor or using `--load-plugin` or similar.

Reply:
In theory, I can drop this commit and hope that we do not do anything more complicated which requires `-load` or any `-l` clang option.